submission/proxy_server.go

```diff
@@ -43,8 +44,21 @@ func NewProxyServer(logListPath string, dBuilder DistributorBuilder, reqTimeout
 }
 
 // Run starts regular Log list updates. Blocks until initialization happens.
```

Comment:
Is this comment accurate? There seems no blocking on initialization. Please fix the comment or the behavior.
While you are here, could you please also comment on how to stop this running? E.g.:
```golang
// Run starts periodic background Log list updates, which terminate when the
// passed-in context is canceled.
```

Reply:
Technically it does block. Meaning it does not return until init happens (internal Run is finished).

Reply:
Not sure I understand what is meant by initialization then. In `Proxy.Run` there is `init` bit which is updated asynchronously, and it doesn't seem that this code blocks on it.

Reply:
Ah, didn't notice the  `<-s.p.Init` line below, ignore the blocking behavior comment.